docs/application/dotnet/samples/watch/overview.md

```diff
@@ -1,23 +1,26 @@
 # Watch
 
+(M) : for mobile profile, (W) : for wearable profile, (T) : for TV profile
 <table>
 	<tbody>
 		<tr>
 			<td><img alt="" height="180" src="media/wambientwatch.png" width="180"/></td>
 			<td>
 			<p><strong>(W) AmbientWatch</strong> [In progress]</p>
-			<p>The AmbientWatch application demonstrates how to make a watch application that supports ambient mode in Tizen wearable device.
-			To extend battery life, a watch application can display a limited UI in ambient mode by detecting ambient mode changes.<br>
+			<p>The AmbientWatch application demonstrates how to create a watch application that supports ambient mode in Tizen wearable device.
+			To extend battery life, a watch application can display a limited UI in the ambient mode by detecting ambient mode changes.<br>
 			</p>
-			<p>First of all, <i>"ambient-support"</i> attribute should be set to true in <b>tizen-manifest.xml</b> file as follows:<br>
+			<p>In <b>tizen-manifest.xml</b> file, set <code>"ambient-support"</code> attribute to true:<br>
```

Comment:
Can we avoid "". It is already in the code tag. Please verify.

Reply:
done